kokoro/macos/package.sh

```diff
@@ -70,7 +70,10 @@ iconutil -c icns -o GAPID.app/Contents/Resources/GAPID.icns GAPID.iconset
 pip install --upgrade --user dmgbuild pyobjc-framework-Quartz
 cp "$SRC"/background*.png .
 cp "$SRC/dmg-settings.py" .
-~/Library/Python/2.7/bin/dmgbuild -s dmg-settings.py GAPID gapid-$VERSION-macos.dmg
+# This path to dmgbuild (installed via pip --user) may change with Kokoro MacOS
+# updates. Value of 'site.USER_BASE' can give a hint of where to look, but
+# beware that default Python version may be different from the pip one.
+/Users/kbuilder/Library/Python/3.7/bin/dmgbuild -s dmg-settings.py GAPID gapid-$VERSION-macos.dmg
```

Comment:
use `~/Library...` so that the script at least has a chance of working outside of the buildbots.

Reply:
Done